backend/app/api/api_v1/endpoints/scim.py

```
@@ -0,0 +1,555 @@
+        "totalResults": len(rows),
+        "startIndex": 1,
+        "itemsPerPage": len(rows),
+        "Resources": [_scim_user_to_dict(user, workspace) for user in rows],
+    }
+
+
+@router.post("/Users", status_code=status.HTTP_201_CREATED)
+async def create_scim_user(
+    payload: ScimUserWrite,
+    request: Request,
+    db: Session = Depends(get_db),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)